src/components/Dashboard/TodoList.vue

```diff
@@ -293,8 +252,8 @@
                     <v-list class="actions__more-actions">
                       <v-list-item
                         v-if="businessId"
-                        id="btn-delete-draft-alt"
-                        @click="confirmDeleteDraft(item)"
+                        id="btn-draft-delete"
+                        @click.stop="confirmDeleteDraft(item)"
```

Comment:
Bugfix: this stops the action from bubbling up to various parents (which eventually cause this panel to open or close).